src/main/kotlin/no/digdir/informasjonsforvaltning/fdk_dataset_harvester/controller/DatasetsController.kt

```diff
@@ -41,16 +41,26 @@ open class DatasetsController(
         }
     }
 
-    @DeleteMapping("/{id}")
+    @PostMapping("/{id}/remove")
```

Comment:
Hva er forskjellen mellom remove og purge? Synes ikke POST /id/remove og DELETE /id virker logisk slik det er nå.

Reply:
Hadde behold DELETE /id for remove og POST /id/purge kanskje?

Reply:
remove logikken vil bare oppdatere meta-objektet til å inkludere removed=true, så den utfører egentlig ikke en sletting. Det er purge logikken som faktisk fjerner ressursen, så jeg vil si det er mer korrekt.

Med slik jeg satte det opp nå:
POST til `/id/remove` -> ressursen fjernes fra søket på Data.norge, men det vil fortsatt være mulig å hente ressursen fra høsteren via GET til `/id` her på høsteren

DELETE til `/id` -> fjerner alt fra høsteren ang. ressursen, så etter dette vil GET til `/id` faktisk gi 404